Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile-upstream:master
 
-ARG RUNC_VERSION=v1.1.5
+ARG RUNC_VERSION=v1.1.6
```

Comment:
Should we have the Dockerfile changes separate from the vendor changes? They should (probably, and at least for runc) have no direct relation with each-other

The Dockerfile is the binary version to test against, and the vendor is "what dependency version" to use to build BuildKit itself.

Having them separate (separate PRs?) also allows verifying that the binary updates work separate from updating buildkit itself (and could help catching regressions / backward-compatibility breaking changes) in any of these)

Reply:
Thank you for the comment. Separated go.mod changes to other PRs

- stargz-snapshotter: https://github.com/moby/buildkit/pull/3792
- runc: https://github.com/moby/buildkit/pull/3793